### DISCUSSION TITLE
Moved

### DISCUSSION THREAD

**Discussion by twister21** [community/community#60763] [Other Feature Feedback, Questions, & Ideas]
This discussion has been moved to another place.

**Comment by GhostOf0days** [comment]
I'm sorry, but this is just not how business works and would not only never happen but would also hurt innovation and consumers.

**Reply by twister21** [reply]
What kind of “innovations” are these platforms currently releasing? 
And how would it hurt consumers to have a more integrated system that allows everyone free access to data?
And what do these requirements have to do with business?

**Reply by GhostOf0days** [reply]
A lot. Also, it's simply not how business works.

**Reply by twister21** [reply]
I've hoped for some more advanced thoughts.

**Reply by GhostOf0days** [reply]
Yes, but this should be an obvious answer. It's not how the world or business works.

**Reply by GhostOf0days** [reply]
It also would most certainly be not feasible financially or legally.

**Reply by twister21** [reply]
It's exactly how the world & programming work. Programmers & nature actually try to come up with the best way to do something. In this case, it's open-source, open-data & well-connected platforms which aren't controlled by a single entity, but by their contributors & users.

**Reply by GhostOf0days** [reply]
It’s not how the world works in the slightest. This should be common sense.

**Reply by twister21** [reply]
You actually have to provide evidence to the contrary. "[...] should be common sense" does not meet this requirement.
